appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -196,11 +196,18 @@ export class SiteRuntimeStep extends AzureWizardPromptStep<IAppServiceWizardCont
         ];
     }
 
-    private sortQuickPicksByRuntime(runtimeItems: IAzureQuickPickItem<ILinuxRuntimeStack>[], recommendedRuntimes: string[]): IAzureQuickPickItem<ILinuxRuntimeStack>[] {
-        function getPriority(item: IAzureQuickPickItem<ILinuxRuntimeStack>): number {
```

Comment:
I don't understand why you got rid of the `getPriority` function. It reduces the amount of copied code

Reply:
Because of tslint. It said "code doesn't need a wrapper, just call it directly"